packages/@aws-cdk/core/test/test.stack.ts

```diff
@@ -44,14 +44,15 @@ export = {
 
     stack.templateOptions.description = 'StackDescription';
     stack.templateOptions.templateFormatVersion = 'TemplateVersion';
-    stack.templateOptions.transform = 'Transform';
+    stack.templateOptions.transform = 'DeprecatedField';
```

Comment:
As a general rule of thumb, I always rather see an additional test instead of modification of an existing test.

Reply:
Yup but this particular code path is stimulated twice in this test suite... And I'm changing only one of them 🤷🏻‍♂️